provisioning/transport/http/src/http.ts

```diff
@@ -192,11 +192,21 @@ export class Http extends EventEmitter implements X509ProvisioningTransport, Tpm
         debug('error executing PUT: ' + err.toString());
         callback(err, result, response);
       } else {
-        debug('PUT response received:');
+        debug('PUT result received:');
         debug(JSON.stringify(result));
+        debug('PUT response.headers received:');
+        debug(JSON.stringify(response.headers));
+        let retryAfter: number = this._config.pollingInterval;
+        /*Codes_SRS_NODE_PROVISIONING_HTTP_06_011: [ If the `registrationRequest` response contains the `header` `retry-after`, it will be interpreted as the number of seconds that should elapse before the next attempted operation.] */
+        if (response.headers && response.headers['retry-after']) {
```

Comment:
>retry-after [](start = 50, length = 11)

is this always lowercase? i thought the swagger file had it PascalCased? or am i confused?

Reply:
Get in line.  The casing was all over the map (pardon the pun).  I tried to be consistent in my code and will switch when they deploy a new release.  What is particularly amazing is that lint/tsc let me include a string constant to reference the property.  Will fix to not have this hard coded.  

---
In reply to: [282200698](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282200698) [](ancestors = 282200698)

Reply:
I have been assured that it will be retry-after.

---
In reply to: [282229287](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282229287) [](ancestors = 282229287,282200698)